R/show.R

```diff
@@ -254,6 +254,21 @@ setMethod("show", "CrunchLogicalExpr", function (object) {
     invisible(object)
 })
 
+showMultitable <- function (x) {
+    out <- paste("Multitable", dQuote(name(x)))
+    
+    # TODO: check variable types to alert users as needed
+    # * check if numerics are binned
+    # * check for datetime rollup resolutions
```

Comment:
I'm not sure that we care. If they're binned, `formatExpression` will print `bin(var)`, so the standard behavior would be fine.